src/parquet/column/levels-test.cc

```diff
@@ -107,10 +150,9 @@ TEST_F(TestLevels, TestEncodeDecodeLevels) {
     parquet::Encoding::type encoding = encodings[encode];
     // BIT_PACKED requires a sequence of atleast 8
     if (encoding == parquet::Encoding::BIT_PACKED) min_repeat_factor = 3;
-
+    int num_levels_per_width = ((2 << max_repeat_factor) - (1 << min_repeat_factor));
```

Comment:
Can we remove this slightly esoteric variable altogether and the `num_levels` variable too? Instead, use `push_back` to append generated levels to `input_levels`